bundles/CoreBundle/Command/InternalNewsletterDocumentSendCommand.php

```diff
@@ -145,20 +145,21 @@ protected function doSendMailInSingleMode(Model\Document\Newsletter $document, A
         $limit = $fifth > 10 ? 10 : ($fifth < 3 ? 3 : intval($fifth));
         $offset = 0;
         $hasElements = true;
+        $index = 1;
 
         while ($hasElements) {
             $tmpStore = Model\Tool\TmpStore::get($sendingId);
 
             $data = $tmpStore->getData();
 
-            Logger::info('Sending newsletter ' . $hasElements . ' / ' . $totalCount. ' [' . $document->getId(). ']');
-
             $data['progress'] = round($offset / $totalCount * 100, 2);
             $tmpStore->setData($data);
             $tmpStore->update();
 
             $sendingParamContainers = $addressAdapter->getParamsForSingleSending($limit, $offset);
             foreach ($sendingParamContainers as $sendingParamContainer) {
+                Logger::warn('Sending newsletter ' . $index . ' / ' . $totalCount. ' [' . $document->getId(). ']');
```

Comment:
Why severity `warning`?

Reply:
Sorry, I'm super bussy this week.
The issue with the warning is, that an Logger::info isn't logging to the newsletter-sending-output.log.
But it is essential for error tracking, especially related to customers, that you know exactly which customers got the newsletter and which ones not. But maybe I'm missing the point how to partial change the logging to info-Level..

Reply:
Ok, thanks for clarifying! 
Please add a comment there explaining why we're using `warning` so that it doesn't get changed in the future.